MenuExample/Controller/MenuViewController.swift

```diff
@@ -19,6 +19,10 @@ class MenuViewController: UITableViewController {
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
 
+//      For some reason iOS9 ignore background color for this VC when it set from storyboard. With white background animation looks ugly...
+//      TODO: Try to remove it after iOS9 release.
```

Comment:
I've deleted this line and backgroundColor of VC is still clearColor. Maybe it was a xcode bug.